templates/android/library/src/main/java/io/appwrite/Query.kt.twig

```diff
@@ -16,6 +16,22 @@ class Query {
     
     fun search(attribute: String, value: String) = Query.addQuery(attribute, "search", value)
 
+    fun isNull(attribute: String) = "isNull(\"${attribute}\")"
+
+    fun isNotNull(attribute: String) = "isNotNull(\"${attribute}\")"
+
+    fun between(attribute: String, start: Int, end: Int) = Query.addQuery(attribute, "between", listOf(start, end))
+
+    fun between(attribute: String, start: Double, end: Double) = Query.addQuery(attribute, "between", listOf(start, end))
+
+    fun between(attribute: String, start: String, end: String) = Query.addQuery(attribute, "between", listOf(start, end))
```

Comment:
how does between with string work

Reply:
It uses the letters like a numerical value, e.g. B is between A and C. Similarly, Boat is between Apple and Cat. 

If multiple letters match it will check the following letter. E.g. Boat is between Bat and Cat.